File: Grocery App/GroceryManagerApp.js
const prompt = require("prompt-sync")({sigint: true});

// Functions
function AddItem(item, cart){
	cart.push(item);
	let message = "Item added successfully";
	return message;	
}
function RemoveItem(item, cart){
	cart.pop(item);
	let message = "Item remove successfully";
	return message;	
}


let cart = [];

while (true){
	console.log("----------------------------------- \nWelcome to The Grocery Store \n1. Add item \n2. Remove item \n3. Show all available items \n----------------------------------");
	let userInput = prompt("Make a selection: ");
	if (userInput == 1){
		let userInput = prompt("What item are you adding: ");
		let AddingItem = AddItem(userInput, cart);
		console.log("'%s' added successfully",userInput);
	}
	if (userInput == 2){
		let userInput = prompt("What item are you removing: ");
		let RemovingItem = RemoveItem(userInput, cart);
		console.log("'%s' removed successfully",userInput);
	}
	if (userInput == 3){
		console.log("Available items in the cart:")
		for (let count = 0; count < cart.length; count++){
			console.log("%d. %s", count+1, cart[count]);
		}
	}
	if (userInput != 1 && userInput != 2 && userInput != 3){
		console.log("Invalid input, please select from the options")
	}
} // end of while loop

File: Grocery App/grocery_manager_app.py
from app_funtions import *

def main_menu():
	main_menu = """
	Welcome to The Grocery Store
	1. Add item
	2. Remove item
	3. Show all available items
	"""
	print(main_menu)

cart = get_cart(cart = [])

while True:
	main_menu()
	user_input = input("Make a Selection: ")
	match user_input:
		case "1": 
			user_input = input("Enter an item name: ")
			print(add_item(user_input, cart))
		case "2": 
			user_input = input("Enter an item name: ")
			print(remove_item(user_input, cart))

		case "3":
			print("Available items")
			count = 0
			for item in cart:
				count +=1
				print(f"{count}. {item}")